internal/host/validator.go

```diff
@@ -357,9 +359,33 @@ func isDiskEncryptionEnabledForRole(encryption models.DiskEncryption, role model
 	}
 }
 
+func (v *validator) getDiskEncryptionForDay2(host *models.Host) (*types.Luks, error) {
+	var response models.APIVipConnectivityResponse
+	if err := json.Unmarshal([]byte(host.APIVipConnectivity), &response); err != nil {
+		// APIVipConnectivityResponse is not available yet - retrying.
```

Comment:
Why does an error here means that `APIVipConnectivityResponse` is unavailable? can't it mean that we just failed to unmarshal?

Reply:
So as we're using the same object in the service and agent it shouldn't fail on syntax and such. But it might takes a while till the response is available (as it invoked by apivip_check validation).